### PR TITLE
Add some missing dartdocs

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie @HansMuller

File: packages/flutter/lib/src/foundation/serialization.dart [hunk 1]
```diff
@@ -90,68 +102,84 @@ class WriteBuffer {
...
   /// Creates a [ReadBuffer] for reading from the specified [data].
   ReadBuffer(this.data) {
     assert(data != null);
   }
 
+  /// The underlying data being read.
+  final ByteData data;
+
+  /// The position to read next.
+  int position = 0;
```
Comment (Hixie) [inline packages/flutter/lib/src/foundation/serialization.dart]: should this be private? seems weird to be able to twiddle it at will...

File: packages/flutter/lib/src/rendering/layer.dart [hunk 2]
```diff
@@ -522,7 +522,12 @@ class BackdropFilterLayer extends ContainerLayer {
   }
 }
 
+/// A composited layer that uses a physical model to producing lighting effects.
```
Comment (Hixie) [inline packages/flutter/lib/src/rendering/layer.dart]: this should explain "physical model"

File: packages/flutter/lib/src/widgets/framework.dart [hunk 3]
```diff
@@ -2564,7 +2564,7 @@ abstract class Element implements BuildContext {
       owner.scheduleBuildFor(this);
     if (hadDependencies)
       didChangeDependencies
-      
+
       ();
```
Comment (Hixie) [inline packages/flutter/lib/src/widgets/framework.dart]: looks like this is still a bit busted

**Comment by Hixie** [comment]
LGTM

File: packages/flutter/lib/src/foundation/serialization.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter/lib/src/foundation/serialization.dart]: Done.

File: packages/flutter/lib/src/rendering/layer.dart (hunk 2 above)
Comment (abarth) [inline packages/flutter/lib/src/rendering/layer.dart]: done.

**Pull request merged**